### PR TITLE
Add nano release script

### PR DESCRIPTION
**Pull request by yangw1234**

### CODE REVIEW

**Comment by yangw1234** [comment]
@Le-Zheng will this work?

**Comment by Le-Zheng** [comment]
Ok, you can use this Jenkins job to test pr: http://10.112.231.51:18888/view/ZOO-PR/job/ZOO-PR-Release-Nano-Python/

File: python/nano/dev/release.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+RUN_SCRIPT_DIR=$(cd $(dirname $0) ; pwd)
+echo $RUN_SCRIPT_DIR
+NANO_DIR="$(cd ${RUN_SCRIPT_DIR}/../; pwd)"
+echo $NANO_DIR
+
```
Comment (Le-Zheng) [inline python/nano/dev/release.sh:22]: It may need to pack source code "python setup.py sdist" before running bdist_wheel.

**Comment by yangw1234** [comment]
http://10.112.231.51:18888/view/ZOO-PR/job/ZOO-PR-Release-Nano-Python/4/

**Pull request merged**